styles.css

```diff
@@ -403,3 +377,93 @@ h6 {
   justify-content: space-between;
   margin-top: 30px;
 }
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  width: 100%;
+  height: 779px;
+  background: #6070ff;
+  margin-top: 40px;
+  border-radius: 75px 0 0 0;
+  background-image: url(./media/contact\ form\ shapes-mobile.svg);
+}
```

Comment:
- [ ] The background image is repeating and not perfectly positioned. Kindly try setting the `background-repeat` attribute to one that makes the image not repeat and position the image so it can look like the Figma template.
 -Your design:
<img width="350" alt="image" src="https://user-images.githubusercontent.com/61919617/193849194-3cead726-a4b7-4e0c-aa2c-01932863b569.png">